addWinner.php

```diff
@@ -0,0 +1,19 @@
+<?php
+?>
+
+<div class="addWinner">
+    <form action="addWinner.php" method="post">
```

Comment:
if you want this form to sub mit data back to this file then you can just leave the action attribute out entirely.